appengine-java11/tasks/pom.xml

```diff
@@ -45,10 +45,21 @@ Copyright 2019 Google LLC
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-tasks</artifactId>
-      <version>1.6.0</version>
-      <scope>provided</scope>
+      <version>1.3.0</version>
```

Comment:
Why the roll back?

Reply:
The library is broken, hence the added tests to catch this next time.